fuzz/uufuzz/Cargo.toml

```diff
@@ -6,7 +6,7 @@ repository = "https://github.com/uutils/coreutils/tree/main/fuzz/uufuzz"
 version = "0.1.0"
 edition.workspace = true
 license.workspace = true
-
+publish = false
```

Comment:
the goal is to publish it for other projects :)

Reply:
hehe, thx, fixed